src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php

```diff
@@ -14,17 +14,22 @@
 use Sulu\Bundle\WebsiteBundle\Resolver\ParameterResolverInterface;
 use Sulu\Component\Webspace\Analyzer\RequestAnalyzerInterface;
 use Symfony\Bundle\TwigBundle\Controller\ExceptionController as BaseExceptionController;
+use Symfony\Component\Debug\Exception\FlattenException;
```

Comment:
the problem here is not that we use it the problem is the typehint of the extended class!